app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -13,7 +13,7 @@
   <FrameLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/question_player_background">
+    android:background="@color/question_player_activity_layout_background_color">
```

Comment:
Please update all colors to follow newer convention having file name in the color declaration.

Reply:
> Please update all colors to follow newer convention having file name in the color declaration.

Can you please elaborate how should I update the name here.
I have already added the file name in the beginning of the color name.